tests/factories/__init__.py

```diff
@@ -0,0 +1,27 @@
+import factory
+from django.utils import timezone
+
+faker = factory.Faker
+
+
+class PlayerFactory(factory.django.DjangoModelFactory):
+
+    nickname = faker("name")
+    motto = "mottos can be the same"
+
+    class Meta:
+        model = "mars_api.Player"
+
+
+class GameFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = "mars_api.Game"
+
+
+class PlayerScoreFactory(factory.django.DjangoModelFactory):
+
+    player = factory.SubFactory(PlayerFactory)
```

Comment:
Looks good! Can't pick up on anything, even small, really

Reply:
just noticed - any particular reason this isn't a `factories.py` module but a folder and a `__init__` file?

Reply:
Well, I've followed the _pytest-factoryboy_ [doc tutorial ](https://pytest-factoryboy.readthedocs.io/en/latest/):
![image](https://user-images.githubusercontent.com/34490649/89731852-5a8b5980-da42-11ea-9498-711a6eaba909.png)

Doesn't make much difference now, but later on might be nice to have a separate folder